tests/foreman/longrun/test_inc_updates.py

```diff
@@ -223,7 +223,7 @@ def get_applicable_errata(repo):
         )
 
     @run_only_on('sat')
-    def test_api_inc_update_noapply(self):
+    def test_positive_api_inc_update_noapply(self):
```

Comment:
I suggest to use `test_positive_noapply_api` as class name already meant `incremental update` operation